src/AzureAuth/CommandMain.cs

```diff
@@ -176,37 +174,17 @@ public CommandMain(CommandExecuteEventData eventData, ITelemetryService telemetr
         public double Timeout { get; set; } = GlobalTimeout.TotalMinutes;
 
         /// <summary>
-        /// Gets or sets the cache file name. Only available on Windows.
+        /// Gets the cache file name. Only available on Windows.
         /// </summary>
-        [Option(CacheOption, "Override the default cache file location. This option is only available on Windows.", CommandOptionType.SingleValue, ShowInHelpText = false)]
-        [LegalFilePath]
         public string CacheFilePath
         {
             get
             {
-                // Check command parameter first.
-                if (!string.IsNullOrEmpty(this.cacheFilePath))
-                {
-                    return this.cacheFilePath;
-                }
-
-                // Check environment variable.
-                string envCacheFile = this.env.Get(EnvVars.Cache);

```

Comment:
I think we should still be able to provide cache file option through env variables.

Reply:
There's 2 reasons I don't think we should do that:
1. We no longer hve any specific use case for doing this.
2. The main motivation for this was cache file sharing -> this is not a recommended way to achieve SSO and should be discouraged. Consumers really should know nothing about the cache file - it's an implementation of AzureAuth and removing this option all together allows us to change that implementation as needed, at will, without breaking potential unknown dependencies.